tests/plugins/balloonpanel/_helpers/tools.js

```diff
@@ -25,10 +25,16 @@ var balloonTestsTools = {
 		}
 	},
 
-	assertMoveTo: function( moveMethod, expectedX, expectedY, maxX, maxY ) {
+	assertMoveTo: function( moveMethod, expectedX, expectedY, maxX, maxY, shouldRound ) {
```

Comment:
This method became highly unreadable. It'd be better to refactor it to use `options` object instead of 6 independent parameters.

Additionally some API docs would be nice.